src/framework/global/async/notifylist.h

```diff
@@ -22,15 +22,133 @@
 #ifndef MU_ASYNC_NOTIFYLIST_H
 #define MU_ASYNC_NOTIFYLIST_H
 
-#include "thirdparty/deto_async/async/notifylist.h"
-#include "thirdparty/deto_async/async/changednotify.h"
+#include <vector>
+#include <cassert>
+
+#include "thirdparty/kors_async/async/changednotify.h"
+
+#include "asyncable.h"
 
 namespace mu::async {
 template<typename T>
-using NotifyList = deto::async::NotifyList<T>;
+using ChangedNotifier = kors::async::ChangedNotifier<T>;
 
 template<typename T>
-using ChangedNotifier = deto::async::ChangedNotifier<T>;
+using ChangedNotify = kors::async::ChangedNotify<T>;
+
+template<typename T>
+class NotifyList : public std::vector<T>
```

Comment:
Could you please explain why it isn't in kors_async?

Reply:
I have doubts about its need for a general lib, that is, this is a special case; we can do a lot of similar things for yourself, for a project.

Reply:
There may be many questions there, for example, why a vector, if a vector, then why the name of the class List, isn’t this confusing, etc.
For a specific project, this is ok, but for a general project, these questions need to be answered.